scripts/installer.sh

```diff
@@ -280,6 +280,14 @@ main() {
 				echo "services.tailscale.enable = true;"
 				exit 1
 				;;
+			bazzite)
+				echo "Bazzite comes with Tailscale installed by default."
+				echo "Please enable Tailscale by running the following commands as root:"
+				echo
+				echo "systemctl enable --now tailscaled"
```

Comment:
I believe Bazzite recommends using their [ujust tool](https://docs.bazzite.gg/Installing_and_Managing_Software/ujust/) for this: `ujust enable-tailscale`

Reply:
Done